modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-renderer/src/main/resources/META-INF/resources/js/index.js

```diff
@@ -56,3 +56,8 @@ export {
 	FormFieldSettings,
 	FormFieldSettingsAdapter,
 } from './custom/form/FormFieldSettings.es';
+
+// Schema
+
+export {DataDefinitionSchema} from './core/schema/DataDefinition.es';
+export {DataLayoutSchema} from './core/schema/DataLayout.es';
```

Comment:
It would be good not to expose this because these are internal methods of this module and we probably don't want people to know about it 😁